### PR TITLE
Add chirashi@5.2.2 with git auto-update

### LINKED ISSUE #10143: [Request] chirashi
**Library name:** chirashi
**Git repository url:** [https://github.com/chirashijs/chirashi](https://github.com/chirashijs/chirashi)
**npm package url(optional):** [npmjs.com/package/chirashi](http://npmjs.com/package/chirashi)
**License(s):** MIT
**Official homepage:** [chirashijs.org](http://chirashijs.org)
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by clairetsai818**
close #10143, cc @chirashijs

Pull request for issue: #10143 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/chirashijs/chirashi
 * Official website (optional, not the repository): chirashijs.org
 * NPM package url (optional): npmjs.com/package/chirashi
 * License and its reference:  MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 36
   - Count of watchers: 4
   - Count of forks: 4
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 1362
   - Downloads in the last month: 1549

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/chirashi/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "keywords": [
+    "DOM",
+    "events"
+  ],
+  "npmName": "chirashi",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/chirashi/package.json]: You should exclude `.gitkeep` and `chirashi.min.js.gz`
Ref: https://github.com/isaacs/node-glob#glob-primer

**Comment by clairetsai818** [comment]
@PeterDaveHello I've corrected the path info. Please help me review this, thanks.

File: ajax/libs/chirashi/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "keywords": [
+    "DOM",
+    "events"
+  ],
+  "npmName": "chirashi",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "chirashi.!(*gz)"
```
Comment (PeterDaveHello) [inline ajax/libs/chirashi/package.json]: Let's keep the `**/` prefix?

**Comment by clairetsai818** [comment]
@PeterDaveHello I kept the prefix `**/`. Please help me review this, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 unless auto-update can't work on the old versions, you'll only need to add the latest version.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've removed old versions. Please help me review this, thanks.

**Comment by clairetsai818** [comment]
@PeterDaveHello I added homepage info in package.json. Please help me review this.

**Comment by PeterDaveHello** [comment]
@clairetsai818 where is the protocol in the url?

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me revise this, thanks!

**Comment by PeterDaveHello** [comment]
Which part you want me to revise for you?

**Comment by clairetsai818** [comment]
@PeterDaveHello I've revised the homepage info. Please help me review this, thanks!

**Comment by clairetsai818** [comment]
@PeterDaveHello I've updated to v5.2.2, please help me review this. Thanks!

**Comment by clairetsai818** [comment]
@PeterDaveHello I've updated to v5.2.2, please help me review this. Thanks!

File: ajax/libs/chirashi/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "keywords": [
+    "DOM",
+    "events"
+  ],
+  "npmName": "chirashi",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/chirashi.!(common.js|mjs)"
```
Comment (PeterDaveHello) [inline ajax/libs/chirashi/package.json]: Where is previous `gz` pattern? So don't we need the `.common.js`?

**Comment by clairetsai818** [comment]
@PeterDaveHello I revised the file map, please help me review this. Thanks!

**Comment by clairetsai818** [comment]
@PeterDaveHello I fixed the file map. Please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
@clairetsai818 Don't you minify chirashi.common.js?

**Comment by clairetsai818** [comment]
@PeterDaveHello I minified chirashi.common.js. Please help me review this, thanks!

**Pull request merged**